powerauth-java-http/src/main/java/io/getlime/security/powerauth/http/validator/PowerAuthEncryptionHttpHeaderValidator.java

```diff
@@ -69,7 +69,7 @@ public static void validate(PowerAuthEncryptionHttpHeader header) throws Invalid
         final String version = header.getVersion();
         if (version == null || version.isEmpty()) {
             throw new InvalidPowerAuthHttpHeaderException("POWER_AUTH_ENCRYPTION_VERSION_EMPTY");
-        } else if (Arrays.asList(VERSIONS).contains(version)) {
+        } else if (!Arrays.asList(VERSIONS).contains(version)) {
```

Comment:
What about to introduce a constant `Set<String> VERSIONS`?

Reply:
```java
/**
   * Admissible protocol versions in the header.
   */
  private static final String[] VERSIONS = { "1.0" };
```

Isn't it better to change javadoc or change the `VERSIONS` to `{"2.0", "2.1", "3.0", "3.1"}`

Reply:
Fixed with the latest improvements.